### PR TITLE
[TCE] convert static Quebecois to dynamic page

### PR DESCRIPTION
**Pull request by astronaut-wannabe**
Pretty cool how fast this went. I:

1. ran the scraper script to generate the `yml` from the static file
2. ran the generator to create the wiring
3. just copied the contents of `public/tce/quebecois/modify.css` into the `app/assets/stylesheets/to_change_everything/_quebecois.scss` file

and it looks like it is pretty much done visually. I still want to

- [x] delete the `public/` assets (left in place to run old and new side-by-side in staging
- [x] reduce some of the CSS copied from `modify.css`

## demo

- static version: https://crimethinc.com/tce/quebecois
- dynamic version:  https://crimethinc-staging-pr-948.herokuapp.com/tce/quebecois

### CODE REVIEW

**Comment by astronaut-wannabe** [comment]
@veganstraightedge the quebecois page in production still has an "order in bulk" link (which is just a `mailto:` link to `anarchy@submedia.tv`

I know you removed a bunch of "order in bulk" links recently and want to know if I should leave this one in place?
screen shot 2018-11-20 at 10 33 16 am

**Comment by just1602** [comment]
Is this normal there is a huge white space a the end of the dynamic page. Otherwise it's really clean !

**Comment by astronaut-wannabe** [comment]
@just1602, It's because there is no "the secret is to begin" panel in this version. Looking into how to optionally hide certain panels

**Comment by veganstraightedge** [comment]
@just1602 That blank space is where `The secret is to begin.` usually is.

File: app/assets/stylesheets/to_change_everything/_quebecois.scss [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          app/views/layouts/to_change_everything.html.erb ---*/
+    #lang1  li  > a[href="/tce/quebecois"] {
+        color: #d62d34 !important;
+    }
+
+    .check-quebecois:before {
+        content: '';
+    }
+
+    #leaders, #answering, #liberation, #control, #relationships, #profit {
```
Comment (astronaut-wannabe) [inline app/assets/stylesheets/to_change_everything/_quebecois.scss]: the CSS in this file is mostly just trying to get all of the text in the paragraphs to not overflow into the next section for any of the 2-column layouts

File: app/views/to_change_everything/show.html.erb [hunk 2]
```diff
@@ -33,7 +33,9 @@
     <h1><%= t '.take_flight_html' %></h1>
   </div>
 
-  <div class="spread" id="next">
-    <h1><%= link_to t('.next_text'), t('.next_path') %></h1>
-  </div>
+  <% if t('.next_text').present? %>
```
Comment (astronaut-wannabe) [inline app/views/to_change_everything/show.html.erb]: this skips rendering `the secret is to begin` section if there isn't a translation for it

File: app/assets/stylesheets/to_change_everything/quebecois/_font_overrides.scss [hunk 3]
```diff
@@ -0,0 +1,88 @@
+@import "../common/font_styling";
+
+.title-container h1 {
```
Comment (astronaut-wannabe) [inline app/assets/stylesheets/to_change_everything/quebecois/_font_overrides.scss]: actually, i think most of these are copy/paste from the `common/font_styling`. let me de-dupe and reduce this further

Comment (astronaut-wannabe) [reply app/assets/stylesheets/to_change_everything/quebecois/_font_overrides.scss]: fixed

**Pull request merged**